scripts/toolchains/linux.cmake

```
@@ -1,4 +1,13 @@
 if(NOT _VCPKG_LINUX_TOOLCHAIN)
+
+cmake_minimum_required(VERSION 3.24)

```

if(POLICY CMP0056)
    cmake_policy(SET CMP0056 NEW)
endif()
if(POLICY CMP0066)
    cmake_policy(SET CMP0066 NEW)
endif()
if(POLICY CMP0067)
    cmake_policy(SET CMP0067 NEW)
endif()
if(POLICY CMP0137)
    cmake_policy(SET CMP0137 NEW)
endif()
```suggestion
if(POLICY CMP0056)
    cmake_policy(SET CMP0056 NEW)
endif()
if(POLICY CMP0066)
    cmake_policy(SET CMP0066 NEW)
endif()
if(POLICY CMP0067)
    cmake_policy(SET CMP0067 NEW)
endif()
if(POLICY CMP0137)
    cmake_policy(SET CMP0137 NEW)
endif()```